src/components/ClipboardButton.js

```diff
@@ -8,7 +8,7 @@ import { icon } from 'src/components/icons';
 import { withErrorReporting } from 'src/libs/error';
 import * as Utils from 'src/libs/utils';
 
-export const ClipboardButton = ({ text, onClick, children, ...props }) => {
+export const ClipboardButton = ({ text, onClick, children, size = 16, ...props }) => {
```

Comment:
Suggest calling this `iconSize` (since it's sizing the icon and not the button) and leaving it undefined by default so that it uses the default size defined in `icon`.

```suggestion
export const ClipboardButton = ({ text, onClick, children, iconSize, ...props }) => {
```